it/uk/gov/hmrc/partnershipidentificationfrontend/controllers/RegistrationControllerISpec.scala

```diff
@@ -230,8 +230,8 @@ class RegistrationControllerISpec extends ComponentSpecHelper with AuthStub with
           stubRetrieveSautr(testJourneyId)(status = OK, body = testSautr)
           stubRetrieveCompanyProfile(testJourneyId)(status = OK, body = Json.toJsObject(testCompanyProfile))
           stubRetrieveBusinessVerificationStatus(testJourneyId)(status = OK, body = Json.toJson[BusinessVerificationStatus](BusinessVerificationPass))
-          stubRegisterScottishLimitedPartnership(testSautr, testCompanyNumber, testRegime)(status = OK, body = RegistrationFailed)
-          stubStoreRegistrationStatus(testJourneyId, RegistrationFailed)(status = OK)
+          stubRegisterScottishLimitedPartnership(testSautr, testCompanyNumber, testRegime)(status = OK, body = RegistrationFailed(Some(testRegistrationFailure)))
+          stubStoreRegistrationStatus(testJourneyId, RegistrationFailed(Some(testRegistrationFailure)))(status = OK)
           stubRetrievePartnershipDetails(testJourneyId)(OK, testPartnershipFullJourneyDataJsonRegistrationFailed)
```

Comment:
For what it is worth the definition of "testPartnershipFullJourneyDataJsonRegistrationFailed" is now out of date.